src/main/java/bio/terra/JobService.java

```diff
@@ -0,0 +1,92 @@
+package bio.terra;
+
+import bio.terra.model.JobModel;
+import bio.terra.model.JobModel.StatusEnum;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.FlightState;
+import bio.terra.stairway.Stairway;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class JobService {
+
+    private final Stairway stairway;
+
+    @Autowired
+    public JobService(
+            Stairway stairway
+    ) {
+        this.stairway = stairway;
+    }
+
+    public JobModel mapFlightStateToJobModel(FlightState flightState) {
+        FlightMap inputParameters = flightState.getInputParameters();
+        String description = inputParameters.get(JobMapKeys.DESCRIPTION.getKeyName(), String.class);
+        StatusEnum status = inputParameters.get(JobMapKeys.STATUS_CODE.getKeyName(), StatusEnum.class);
+        String submittedDate = new SimpleDateFormat().format(flightState.getSubmitted());
+        String completedDate = new SimpleDateFormat().format(flightState.getCompleted()
+                .orElse(null)); // TODO this doesn't seem right?
+
+        JobModel jobModel = new JobModel()
+                .id(flightState.getFlightId())
+                .description(description)
+                .status(status)
+                .submitted(submittedDate)
+                .completed(completedDate);
+
+        return jobModel;
+
+    }
+
+    public ResponseEntity<List<JobModel>> enumerateJobs(int offset, int limit) {
```

Comment:
i think we will eventually want to be able to limit the job list we return (we will probably want to the stairway interface to support this as well) like submitted after a specific date or jobs in a certain state. not needed for this PR

Reply:
This is following what Jeremy wrote in the endpoint design spec.